src/app/Services/EmployeeUpdaterService.php

```
@@ -66,46 +55,23 @@ public function execute(
-                        newSalary: $data->salary,
-                        effectiveDate: $data->dateJoin ?? now()
-                    );
+            if ($image) {
+                try {
+                    $profileData['image_path'] = $this->imageUploader->upload($image);
+                } catch (RuntimeException $e) {
+                    throw new RuntimeException("Failed to upload image: " . $e->getMessage());
                 }
+            } else {
+                // keep actual image.
+                $profileData['image_path'] = $existingStudent->profile->image_path ?? null;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$existingStudent'. (undefined)

(UnusedLocalVariable)